lib/engines/content_block_manager/app/services/content_block_manager/get_preview_content.rb

```diff
@@ -44,13 +44,24 @@ def preview_html
     end
 
     def replace_existing_content_blocks(nokogiri_html)
-      existing_content_block_spans(nokogiri_html).each do |span|
+      replace_blocks(nokogiri_html)
+      style_blocks(nokogiri_html)
+      nokogiri_html
+    end
+
+    def replace_blocks(nokogiri_html)
+      content_block_spans(nokogiri_html).each do |span|
         span.replace @content_block_edition.render
       end
-      nokogiri_html
     end
 
-    def existing_content_block_spans(nokogiri_html)
+    def style_blocks(nokogiri_html)
+      content_block_spans(nokogiri_html).each do |span|
+        span["style"] = "background-color: yellow;"
```

Comment:
Would it be better to add a class here and then have the styling in the CSS? That way it'll be easier to make any tweakes further down the line - thinking maybe `content-embed--preview` to stay true to the [BEM way of doing things](https://getbem.com/). Could alter this with something like:

```suggestion
        span["class"] = "#{span["class"]} content-embed--preview"
```

Reply:
oh I meant to put something in the commit about this - i just hit a slight complication with adding a style to an iframe from outside of it, but I think it is possible